extensions/wikia/DesignSystem/ds.built.js

```diff
@@ -0,0 +1,768 @@
+
+
+
+		// if you want to add styles, just require them below and let Webpack do it's magic
+
+
+
+
+		/***/ }),
+	/* 1 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__utils_closest_element__ = __webpack_require__(3);
+		/* harmony export (immutable) */ __webpack_exports__["hookDropdown"] = hookDropdown;
+
+
+		var dropdownClass = 'wds-dropdown';
+		var dropdownToggleClass = 'wds-dropdown__toggle';
+		var dropdownContentClass = 'wds-dropdown__content';
+
+		var activeClass = 'wds-is-active';
+
+		/**
+		 * Close all dropdowns except current
+		 * @param   {Element} dropdownToKeepOpen
+		 * @returns {void}
+		 */
+		function closeEveryDropdown(dropdownToKeepOpen) {
+			var activeDropdowns = document.querySelectorAll('.' + dropdownClass + '.' + activeClass);
+
+			if (dropdownToKeepOpen) {
+				[].forEach.call(activeDropdowns, function (dropdown) {
+					if (dropdown !== dropdownToKeepOpen) {
+						dropdown.classList.remove(activeClass);
+					}
+				});
+			} else {
+				[].forEach.call(activeDropdowns, function (dropdown) {
+					dropdown.classList.remove(activeClass);
+				});
+			}
+		}
+
+		var hookedToBody = false;
+
+		/**
+		 * Function that opens given dropdown and closes anything else
+		 *
+		 * @param   {Element} dropdown
+		 * @returns {void}
+		 */
+		function openDropdown(dropdown) {
+			closeEveryDropdown(dropdown);
+			dropdown.classList.add(activeClass);
+		}
+
+		/**
+		 * Function that hooks to the `<body>` and listens on click to close dropdowns
+		 * @returns {void}
+		 */
+		function hookToBody() {
+			if (hookedToBody) {
+				return;
+			}
+
+			hookedToBody = true;
+
+			document.addEventListener('click', function (ev) {
+				var closest = __webpack_require__.i(__WEBPACK_IMPORTED_MODULE_0__utils_closest_element__["a" /* default */])(ev.target, '.' + dropdownClass);
+				closeEveryDropdown(closest);
+			});
+		}
+
+		/**
+		 * Hook to the element
+		 * @param   {Element} dropdown
+		 * @returns {void}
+		 */
+		function hookDropdown(dropdown) {
+			hookToBody();
+
+			dropdown.querySelector('.' + dropdownToggleClass).addEventListener('click', function () {
+				if (dropdown.classList.contains(activeClass)) {
+					dropdown.classList.remove(activeClass);
+				} else {
+					openDropdown(dropdown);
+				}
+			});
+		}
+
+		/* harmony default export */ __webpack_exports__["default"] = ({
+			hookDropdown: hookDropdown
+		});
+
+		/***/ }),
+	/* 2 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
+		/* harmony export (immutable) */ __webpack_exports__["init"] = init;
+		function init() {
+			console.log('hi from test-component');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)